### DISCUSSION TITLE
what are the steps to use udid registrations to sign uYouPlus IPA?

### DISCUSSION THREAD

**Discussion by ALPHAQUE66** [qnblackcat/uYouPlus#395] [Q&A] [answered]
instead of refreshing the uYouplus every 7 days using sideloadly, i plan to get a certificate and profile at udid registrations for $20 per year which i know it will keep the uYouplus valid for 1 year. The problem is i don't know know to sign the uYouplus IPA  using the certificate and profile i bought from UDID registration, can anyone give me some simple steps how to achieve that? some said need to use https://www.iosappsigner.com/, but i have no idea how to do it, any helps will be much appreciated,  thanks

**Comment by qnblackcat** [comment, marked as answer]
https://www.udidregistrations.com/sign-apps

**Reply by ALPHAQUE66** [reply]
thanks, if i buy the "silver" package, i understand i can't use those online signer like https://www.ipasigner.com/, and i don't have a Mac as well so i can't use [iOS App Signer] stated in the guide you posted. May i know how do i sign the IPA if i am using the "silver" package either online or on a Windows platform? thanks

**Reply by qnblackcat** [reply]
You can sign your IPA online via your web browser using SignTools. The setup is a bit complicated tho, just make sure you follow the tutorial step by step.

https://github.com/SignTools

Windows? I’m not a big fan of Windows so I have no clue.
